pycbc/events/events.py

```diff
@@ -220,6 +220,22 @@ def newsnr_sgveto(snr, bchisq, sgchisq):
     else:
         return nsnr[0]
 
+def newsnr_sgveto_psdvar(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD 
+    variation statistic """
+    nsnr = newsnr_sgveto(snr, bchisq, sgchisq)
+    nsnr = numpy.array(nsnr, ndmin=1)
+    psd_var_val = numpy.array(psd_var_val, ndmin=1)
+    t = numpy.array(psd_var_val >= 1.8, ndmin=1)
+    if len(t) > 0:
+        nsnr[t] = nsnr[t] / numpy.sqrt(psd_var_val[t])
```

Comment:
```
    t = numpy.array(psd_var_val >= 1.8, ndmin=1)
    if len(t) > 0:
        nsnr[t] = nsnr[t] / numpy.sqrt(psd_var_val[t])
```
I think this can be shortened to
```
lgc = psd_var_val >= 1.8
nsnr[lgc] = nsnr[lgc] / numpy.sqrt(psd_var_val[lgc])
```

Reply:
sure, I was just following what was done in other functions. But I've updated the code to this suggestion.